### PR TITLE
Redux reducers and actions

### PR DESCRIPTION
**Pull request by IsmaelMastronardi**
On this branch, I worked on adding Redux to my project.

- Added a store.
- Added a bookSlice, with 2 reducers (addBook and removeBook).
- -added a categoriesSlice with 1 reducer (statusCheck).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 ### Tech Stack <a name="tech-stack"></a>
```
Comment (Munsa1) [inline README.md:32]: - [ ] Great work the app works fine, just to clean your readme file so it's more professional, kindly remove the link to the `Live Demo` section as it is a broken link leading nowhere.
image

- [ ] Kindly ensure the issues link points to this project.
image

- [ ] Kindly fix the link for cloning the project under the getting started section
image

File: src/redux/books/bookSlice.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      const bookItem = {
+        title: payload.title,
+        author: payload.author,
+        // genre: payload.genre,
```
Comment (Munsa1) [inline src/redux/books/bookSlice.js]: - [ ] Kindly remove this commented line of code for a cleaner codebase

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @IsmaelMastronardi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Used Redux Correctly :heavy_check_mark: 
Project Folder Structure Follows Ducks Pattern :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @IsmaelMastronardi 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Understand the concept of state management.
- ✔️ Use store, actions, and reducers in React.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**